op-supervisor/supervisor/backend/l1access/l1_accessor.go

```diff
@@ -159,8 +162,33 @@ func (p *L1Accessor) onFinalized(ctx context.Context, ref eth.L1BlockRef) {
 }
 
 func (p *L1Accessor) onLatest(ctx context.Context, ref eth.L1BlockRef) {
+	// Stop if the block is the same or older than the tip
+	if ref.Number == p.tipHeight && ref.Hash == p.tipHash {
+		p.log.Warn("L1 block is the same as the tip", "ref", ref)
+		return
+	}
+	if ref.Number < p.tipHeight {
+		p.log.Warn("L1 block is older than the tip", "ref", ref)
+		return
+	}
+
+	// Update the tip
 	p.tipHeight = ref.Number
+	p.tipHash = ref.Hash
```

Comment:
Would be nice to track as single `eth.BlockID` attribute, so we never forget to update one but not the other.